lib/generate-suggestions.ts

```diff
@@ -0,0 +1,96 @@
+import fs from "fs-extra";
+import glob from "glob";
+import { parse } from "@babel/parser";
+import traverse from "@babel/traverse";
+
+import { fetchComponents } from "./http/fetchComponents";
+
+async function generateSuggestions() {
+  const components = await fetchComponents();
+  const results: {
+    [compApiId: string]: FindResults;
+  } = {};
+
+  for (const [compApiId, component] of Object.entries(components)) {
+    if (!results[compApiId]) {
+      results[compApiId] = [];
+    }
+    const result = await findTextInJSXFiles(".", component.text);
+    results[compApiId] = [...results[compApiId], ...result];
+  }
+  console.log(JSON.stringify(results, null, 2));
+}
+
+interface Occurence {
+  lineNumber: number;
+  preview: string;
+}
+
+type FindResults = { file: string; occurrences: Occurence[] }[];
+
+async function findTextInJSXFiles(
+  path: string,
+  searchString: string
+): Promise<FindResults> {
+  const result: { file: string; occurrences: Occurence[] }[] = [];
+  const files = glob.sync(`${path}/**/*.+(jsx|tsx)`);
+
+  for (const file of files) {
+    const code = await fs.readFile(file, "utf-8");
```

Comment:
nit: since order doesn't matter, we could technically do this in a `Promise.all` rather than iteratively in order using a for loop -- not sure if there would be any sort of meaningful performance improvements

Reply:
![image](https://user-images.githubusercontent.com/15795866/230449810-016d03c4-fc6e-4dae-a04a-95de0ca0001a.png)